packages/vue-components/src/questions/Question.vue

```diff
@@ -237,13 +218,10 @@ export default {
       return this.type === 'blanks';
     },
     isTextQuestion() {
-      return this.type === 'text' || this.hasInput;
+      return this.type === 'text';
     },
-    isValidQuestionType() {
-      return this.isMcqOrCheckboxQuestion() || this.isBlanksQuestion() || this.isTextQuestion();
-    },
-    isValidTypeAndNotTextWithoutKeywords() {
-      return this.isValidQuestionType() && !(this.isTextQuestion() && !this.keywords);
+    isNotTextWithoutKeywords() {
+      return !(this.isTextQuestion() && !this.keywords);
```

Comment:
This method name is quite confusing with a double negation (and then a triple negation above when we are negating it to be assigned to `const isTextWithoutKeyword` above) 😅 Can we rename this? Maybe `isTextWithoutKeyword` where it can be used directly?